pkg/network/opencontrail/config.go

```diff
@@ -61,7 +61,7 @@ func (c *Config) Parse(args []string) {
 		"OpenContrail API port.")
 	fs.StringVar(&c.PublicNetwork, "public_name", c.PublicNetwork,
 		"External network name.")
-	fs.StringVar(&c.PublicSubnet, "public_net", "",
+	fs.StringVar(&c.PublicSubnet, "public_net", "c.PublicSubnet",
```

Comment:
Ack. I had meant it as -- fs.StringVar(&c.PublicSubnet, "public_net",
c.PublicSubnet,
The default is "" anyways, as c.PublicSubnet is also initialized to ""

Thanks. I will fix it.


On Mon, Jul 6, 2015 at 10:08 AM, Pedro Marques <notifications@github.com>
wrote:

> In pkg/network/opencontrail/config.go
> <https://github.com/Juniper/contrail-kubernetes/pull/13#discussion_r33957279>
> :
>
> > @@ -61,7 +61,7 @@ func (c *Config) Parse(args []string) {
> >  		"OpenContrail API port.")
> >  	fs.StringVar(&c.PublicNetwork, "public_name", c.PublicNetwork,
> >  		"External network name.")
> > -	fs.StringVar(&c.PublicSubnet, "public_net", "",
> > +	fs.StringVar(&c.PublicSubnet, "public_net", "c.PublicSubnet",
>
> This is an error. The string "c.PublicSubnet" is not a subnet prefix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/13/files#r33957279>.
>